docs/sidebars.ts

```diff
@@ -138,6 +138,11 @@ module.exports = {
               "label": "TypeScript SDK Reference",
               "id": "reference/typescript/modules"
             },
+            {
```

Comment:
As an alternative, wdyt of placing this in the "Working with Modules" section?
- For: it's in the module docs section PLUS the current category it's placed in ie. "API and SDK reference" doesn't seem to match, so moving it might make sense?  
- Against: it a reference in the "reference" section, so this _is_ the correct location. Maybe rename the category to just "Reference"?

Reply:
Yeah, I think that sounds good.